io.github.amourspirit.LibrePythonista_PyEditor.yml

```diff
@@ -0,0 +1,62 @@
+app-id: io.github.amourspirit.LibrePythonista_PyEditor
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: cell_edit
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
```

Comment:
what is network for?

Reply:
This app installs my [librepythonista-python-editor](https://pypi.org/project/librepythonista-python-editor/) which is a [PyWebveiw](https://pywebview.flowrl.com/) based Python project.

```
 - --socket=wayland
 - --socket=fallback-x11
 - --share=ipc
 ```

Is required for the PyWebview (webview) window to display.

When I tested `--share=network` is not need so I removed it.